scripts/mcstatus.coffee

```diff
@@ -0,0 +1,48 @@
+# Description:
+#   Report the status of a Minecraft server
+#   Uses http://api.syfaro.net/
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   server_url and server_port, below
+#
+# Commands:
+#   mcstatus - Report the status of the configured minecraft server.
+#
+# Author:
+#   annabunches
+
+default_url = 'mc.wtf.cat'
```

Comment:
can we throw in `process.env.MINECRAFT_HOST || 'mc.wtf.cat'`

and
`process.env.MINECRAFT_PORT || 10070`

:point_down: 

So we don't have to edit the script if the server moves?